snuba/subscriptions/data.py

```diff
@@ -68,3 +78,8 @@ def build_request(
             dataset,
             SUBSCRIPTION_REFERRER,
         )
+
+
+class Subscription(NamedTuple):
+    identifier: SubscriptionIdentifier
```

Comment:
I'd also be ok calling this `key` since it is essentially a composite primary key.